tests/e2e/support/api/davSpaces/spaces.ts

```diff
@@ -37,3 +40,125 @@ export const createFolder = async ({
     subFolder += resource
   }
 }
+
+export const createFolderInsideSpace = async ({
+  user,
+  folderName,
+  spaceName
+}: {
+  user: User
+  folderName: string
+  spaceName: string
+}): Promise<void> => {
+  const paths = folderName.split('/')
+
+  let subFolder = ''
+  for (const resource of paths) {
+    const response = await request({
+      method: 'MKCOL',
+      path: join(
+        'remote.php',
+        'dav',
+        'spaces',
+        await getSpaceIdBySpaceName({ user, spaceType: 'project', spaceName }),
+        subFolder,
+        resource
+      ),
+      user: user
+    })
+
+    checkResponseStatus(response, `Failed while creating a ${folderName} inside space project`)
+    subFolder += resource
+  }
```

Comment:
I think we had a discussion about this. Just a reminder

Reply:
sure i will create a separate PR for that fix.

Reply:
Since you are adding this code, I will suggest adding the code with the correct logic here.